ZEP/instructions/zep0000.md

```diff
@@ -0,0 +1,399 @@
+# Zarr Community Feedback Process - Zarr Enhancement Proposal (ZEP)
+
+--- 
+ 
+## ZEP 0 — Purpose and process
+
+Author: Sanket Verma <svsanketverma5@gmail.com>
+
+Status: Active
+
+Type: Process
+
+Created: 2022-14-03
+
+## What is ZEP?
+
+ZEP stands for Zarr Enhancement Proposal. A ZEP is a design document providing
+information to the Zarr community, describing a modification or enhancement of
+the Zarr storage specification, a new feature for its processes or environment.
+The ZEP should provide specific proposed changes to the Zarr specification and
+a narrative rationale for the specification changes.
+
+We intend ZEPs to be the primary mechanism for evolving the spec, collecting
+community input on major issues and documenting the design decision that has
+gone into Zarr. In addition, the ZEP author is responsible for building
+consensus within the community and documenting dissenting opinions.
+
+Because the ZEPs are maintained as text files in a versioned repository, their
+revision history is the historical record of the feature proposal.
+
+## ZEP audience
+
+The typical primary audience for ZEPs is the developers working on Zarr and its
+various implementations, the Zarr steering council as well as the Zarr
+community.
+
+The broader Zarr community may also choose to use the process to document
+expected feature additions and to manage complex design coordination problems
+that require collaboration across multiple projects.
+
+## ZEP types
+
+> Specification ZEP
+
+Specification ZEPs deal with the changes related to [Zarr Specifications].
```

Comment:
Missing link here

Reply:
These link to https://zarr-specs.readthedocs.io/ for me via the references at the bottom.

Reply:
The links are in the bottom of the document.